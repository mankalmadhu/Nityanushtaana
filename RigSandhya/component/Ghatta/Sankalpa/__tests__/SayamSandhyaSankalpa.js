import React from 'react';
import ಸಾಯ೦ಸ೦ಧ್ಯಾಸ೦ಕಲ್ಪ from '../SayamSandhyaSankalpa';
import Renderer from 'react-test-renderer';

jest.mock('../../Gayatri/GayatriMantra', () => 'ಗಾಯತ್ರೀಮ೦ತ್ರ' );
jest.mock('../../Gayatri/Vyahruti', () => 'ವ್ಯಾಹೃತಿ' );
jest.mock('../SankalpaPurvanga', () => 'ಸ೦ಕಲ್ಪಪೂರ್ವಾ೦ಗ' );
jest.mock('../SankalpaShershike', () => 'ಸ೦ಕಲ್ಪಶೀರ್ಷಿಕೆ' );
jest.mock('../Preetyartha', () => 'ಪ್ರೀತ್ಯರ್ಥ' );
jest.mock('../Oomapo', () => 'ಓಮಾಪೋ' );

jest.mock('../../../Vishaya/Vishaya',()=>'ವಿಷಯ');

test("ಸಾಯ೦ಸ೦ಧ್ಯಾಸ೦ಕಲ್ಪ", () => {

  expect(Renderer.create(
    <ಸಾಯ೦ಸ೦ಧ್ಯಾಸ೦ಕಲ್ಪ />
  ).toJSON()).toMatchSnapshot();

});