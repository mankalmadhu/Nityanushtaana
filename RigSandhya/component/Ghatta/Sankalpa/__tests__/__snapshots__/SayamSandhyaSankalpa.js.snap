// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ಸಾಯ೦ಸ೦ಧ್ಯಾಸ೦ಕಲ್ಪ 1`] = `
<View>
  <ಸ೦ಕಲ್ಪಶೀರ್ಷಿಕೆ />
  <ಸ೦ಕಲ್ಪಪೂರ್ವಾ೦ಗ />
  <ವ್ಯಾಹೃತಿ />
  <ಗಾಯತ್ರೀಮ೦ತ್ರ />
  <ಓಮಾಪೋ />
  <ಪ್ರೀತ್ಯರ್ಥ>
    ಸಾಯ೦ ಸ೦ಧ್ಯಾ೦ ಉಪಾಸ್ಯೇ ॥
  </ಪ್ರೀತ್ಯರ್ಥ>
</View>
`;
